google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb

```diff
@@ -60,14 +63,33 @@ def retrieve_snapshot project, subscription, snapshot_name
     end
 
     it "should be created, updated and deleted" do
-      topic = pubsub.create_topic new_topic_name, labels: labels
+      topic = pubsub.create_topic new_topic_name,
+                                  labels: labels,
+                                  retention: topic_retention
       _(topic).must_be_kind_of Google::Cloud::PubSub::Topic
       topic = pubsub.topic(topic.name)
       _(topic).wont_be :nil?
+
       _(topic.labels).must_equal labels
       _(topic.labels).must_be :frozen?
       topic.labels = {}
       _(topic.labels).must_be :empty?
+
+      _(topic.retention).must_equal topic_retention
+      topic.retention = new_topic_retention
+      _(topic.retention).must_equal new_topic_retention
+
+      subscription = topic.subscribe "#{$topic_prefix}-sub-topic-retention"
+      _(subscription.topic_retention).must_equal new_topic_retention
```

Comment:
This line is still failing in the acceptance tests

Reply:
@hongalex After reverting from the staging endpoint back to production, the subscription is no longer getting its topic's retention value:

```
  1) Failure:
Google::Cloud::PubSub::pubsub::Topic::pubsub#test_0002_should be created, updated and deleted [/workspace/google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb:83]:
Expected: 660
  Actual: nil
```